QUICKSTART.md

```diff
@@ -0,0 +1,88 @@
+# Building the Repository
+
+To build the entire repository and run the application, follow these steps:
+
+1. The 2 main system dependencies to install are:
+
+   - [volta](https://docs.volta.sh/guide/getting-started)
+   - [docker](https://docs.docker.com/get-docker/)
```

Comment:
what about pnpm?

Reply:
Well. I think if you have volta, volta will download the pnpm dependency for you. I think there is some shim pattern such that you can just use `pnpm` command before you even install it on your system since volta will pull it for you if you need. 

I dono maybe I shud explicity state it as a system dependency. But then ppl will be tempted to install their own version and I think if you dont have the VOLTA_FEATURE_PNPM, it might point to your system version rather than the package.json

Reply:
I have added a note inside of it